Lib/ldap/schema/subentry.py

```diff
@@ -474,8 +474,11 @@ def urlfetch(uri,trace_level=0,bytes_mode=None):
     l.unbind_s()
     del l
   else:
-    import urllib,ldif
-    ldif_file = urllib.urlopen(uri)
+    import ldif
+    from ldap.compat import urlopen
+    if not uri.lower().startswith("file://"):
```

Comment:
That's probably not right. A LDAP schema url could be a http or https link, too. The application should pass in a correct URI.